### ISSUE TITLE
[Bug] Storage widget is displaying incorrect values

### ISSUE THREAD

**Issue by AD1995** [MauriceNino/dashdot#182]
### Description of the bug

Storage widget is displaying incorrect values. 
Currently running through Docker Compose with the volume mounted as
  - /:/mnt/host:ro

Hovering over the green bar next to the ATA SSD displays 16700% used and hovering over the blank area next to the WD HDD show's 3% used which is the correct but the bar doesn't display.

https://imgur.com/a/0051ceA

### Relevant log output

```shell
yarn run v1.22.19
$ node .
(node:27) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
listening on *:3001
Multiple default network interfaces found [enp2s0, ], using "enp2s0"
Using network interface "enp2s0"
Using host os version from "/mnt/host/etc/os-release"
Static Server Info: {
  os: {
    arch: 'x64',
    distro: 'Ubuntu',
    kernel: '5.4.0-113-generic',
    platform: 'linux',
    release: '20.04.4 LTS',
    uptime: 2417794.8
  },
  cpu: {
    brand: 'Intel',
    model: 'Xeon® E3-1226 v3',
    cores: 4,
    threads: 4,
    frequency: 3.3
  },
  ram: {
    size: 16698437632,
    layout: [
      { brand: undefined, type: 'Empty', frequency: 0 },
      { brand: 'Hynix/Hyundai', type: 'DDR3', frequency: 1600 },
      { brand: undefined, type: 'Empty', frequency: 0 },
      { brand: 'Micron', type: 'DDR3', frequency: 1333 }
    ]
  },
  storage: {
    layout: [
      {
        device: 'sda',
        brand: 'Western Digital',
        size: 2000398934016,
        type: 'HD'
      },
      {
        device: 'sdb',
        brand: 'ATA',
        size: 500107862016,
        type: 'SSD',
        raidGroup: ''
      }
    ]
  },
  network: {
    interfaceSpeed: 1000,
    speedDown: 0,
    speedUp: 0,
    type: 'Wired',
    publicIp: ''
  },
  gpu: { layout: [] },
  config: {
    port: 3001,
    running_in_docker: true,
    accept_ookla_eula: false,
    use_network_interface: '',
    disable_integrations: false,
    show_host: true,
    use_imperial: false,
    enable_storage_split_view: true,
    always_show_percentages: true,
    widget_list: [ 'os', 'cpu', 'storage', 'ram', 'network' ],
    os_label_list: [ 'os', 'arch', 'up_since' ],
    cpu_label_list: [ 'brand', 'model', 'cores', 'threads', 'frequency' ],
    storage_label_list: [ 'brand', 'size', 'type', 'raid' ],
    ram_label_list: [ 'brand', 'size', 'type', 'frequency' ],
    network_label_list: [ 'type', 'speed_up', 'speed_down', 'interface_speed' ],
    gpu_label_list: [ 'brand', 'model', 'memory' ],
    os_widget_grow: 1.5,
    os_widget_min_width: 300,
    enable_cpu_temps: false,
    cpu_widget_grow: 4,
    cpu_widget_min_width: 500,
    cpu_shown_datapoints: 20,
    cpu_poll_interval: 1000,
    storage_widget_grow: 3.5,
    storage_widget_min_width: 500,
    storage_poll_interval: 60000,
    ram_widget_grow: 4,
    ram_widget_min_width: 500,
    ram_shown_datapoints: 20,
    ram_poll_interval: 1000,
    speed_test_interval: 20,
    network_widget_grow: 6,
    network_widget_min_width: 500,
    network_shown_datapoints: 20,
    network_poll_interval: 1000,
    gpu_widget_grow: 6,
    gpu_widget_min_width: 700,
    gpu_shown_datapoints: 20,
    gpu_poll_interval: 1000,
    override: {
      os: undefined,
      arch: undefined,
      cpu_brand: undefined,
      cpu_model: undefined,
      cpu_cores: undefined,
      cpu_threads: undefined,
      cpu_frequency: undefined,
      ram_brand: undefined,
      ram_size: undefined,
      ram_type: undefined,
      ram_frequency: undefined,
      network_type: undefined,
      network_speed_up: undefined,
      network_speed_down: undefined,
      network_interface_speed: undefined,
      network_public_ip: undefined,
      storage_brands: [],
      storage_sizes: [],
      storage_types: [],
      gpu_brands: [],
      gpu_models: [],
      gpu_memories: []
    }
  }
}
Running speed-test (this may take a few minutes)...
```

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.4.0
Dash: 4.0.0

Cwd: /app
Hash: 8e743a2cc91e76d8e0beb1cf5f8102ea73e97f16
In Docker: true
In Podman: false
In Docker (env): true
Done in 0.33s.
```

### What browsers are you seeing the problem on?

Firefox, Chrome

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hey there, thanks for the issue!

Can you please execute the following command on your system and paste the output here?

```
docker exec CONTAINER_NAME yarn cli raw-data --storage
```

**Comment by AD1995** [comment]
Results are here - https://pastebin.com/D37dvbye

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.1.0

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.
